text/0000-all-builder-buildpackless.md

```diff
@@ -0,0 +1,76 @@
+# Make All Builders Buildpackless
+
+## Summary
+
+Builders that conglomerate buildpacks are unsustainable in the long run as
+there is a layer cap set in place by Docker of 127 layers. We should stop
+adding buildpack to our builder and instead only offer builders that contain
+the stack and lifecycle. 
+
+## Motivation
+
+With the current size of the `full` builder we are already seeing users running
+into issues with exceeding the layer limit of the buildpack when trying to add
+custom buildpacks to their builds. This will only be exacerbated going forward
+with the addition of new buildpacks into the Paketo project. Not only that the
+`full` builders are completely unsustainable in the near future with proposals
+such as the addition of APMs and the restructuring of existing buildpacks which
+will both increase the layer count further.
+
+Currently the purpose of the builders that include buildpacks is to allow first
+time users to quickly plug in a large number of applications into the build
+process and see what comes out. However using these loaded builders is not the
```

Comment:
I think this is still a use case we'd want to support & we should plan how we do that. IMO, we want the first-touch experience with Paketo to be as easy as possible. I think that if we updated our docs/samples/getting started guides with language-specific builder references, the process could still be pretty easy.

The one spot that comes to mind which might not be easy is `pack builder suggest`. I suppose we could ask if we could add the language-specific builders, but even then you'd get a list of 7+ builders and it might be tricky for users to pick one. I don't really know how many users discover us through this mechanism though, or is it more of just a reference for users that already have some familiarity with Paketo, in which case maybe it's not as big of a deal.